Doc/c-api/intro.rst

```diff
@@ -105,6 +105,30 @@ defined closer to where they are useful (e.g. :c:macro:`Py_RETURN_NONE`).
 Others of a more general utility are defined here.  This is not necessarily a
 complete listing.
 
+.. c:macro:: PyMODINIT_FUNC
+
+   Declare an extension module ``PyInit`` initialization function. The function
+   return type is :c:expr:`PyObject*`. The macro declares any special linkage
+   declarations required by the platform, and for C++ declares the function as
+   ``extern "C"``.
+
+   The initialization function must be named :c:func:`!PyInit_name`, where
```

Comment:
Maybe write ``` :samp:`PyInit_{name}` ```? It renders the whole name as a code, but *name* additionally is in italic, as a variable part.

Reply:
I never saw :samp:, let me try.

Reply:
I see, it looks good: https://cpython-previews--109236.org.readthedocs.build/en/109236/c-api/intro.html#c.PyMODINIT_FUNC

![Screenshot 2023-09-14 at 14-55-51 Introduction](https://github.com/python/cpython/assets/194129/668afaf9-32cc-4ba0-a662-ccdac0063100)